bundles/org.openhab.binding.bluetooth.roaming/README.md

```diff
@@ -0,0 +1,41 @@
+# Bluetooth Roaming Adapter
+
+This extension adds support for accessing Bluetooth devices from any other configured adapter via a virtual adapter.
+
+## Supported Things
+
+It defines the following bridge type:
+
+| Bridge Type ID | Description                                                                                                |
+|----------------|------------------------------------------------------------------------------------------------------------|
+| roaming        | A virtual Bluetooth adapter that interacts with Bluetooth devices through their nearest Bluetooth adapter. |
+
+## Channels
+
+Devices which use a `roaming` adapter as their bridge also gain the following channels:
+
+| Channel ID       | Item Type | Description                                                     |
+|------------------|-----------|-----------------------------------------------------------------|
+| adapter-uid      | String    | The thingUID of the adapter that is nearest to this device      |
+| adapter-location | String    | The value of the `Location` specified for the nearest adapter    |
```

Comment:
To me as a reader it is completely unclear what this "Location" string will contain. Would be nice to have some better explanation for it.

Reply:
Will do